### DISCUSSION TITLE
G-Codes larger than 100mb

### DISCUSSION THREAD

**Discussion by pitchforknumb** [Guilouz/Creality-Helper-Script-Wiki#547] [Issues]
Hey all.  I've run into an issue where I can't upload a file from Orca Slicer that is larger than 100mb directly to my printer.  It gives me a HTTP 502 error.  I tried downloading the file to my desktop and uploading it in Fluidd and Mainsail that way, but both programs failed uploading.  I then tried uploading it into Creality Print and it uploaded it just fine on that program.  Is there a workaround for this where I can send the files straight from Orca Slicer?
![502 error](https://github.com/Guilouz/Creality-Helper-Script-Wiki/assets/94194418/01a20197-5ca5-439c-866a-3df244833f65)

**Comment by Guilouz** [comment]
Check moonraker config with max_upload_size parameter.

**Reply by pitchforknumb** [reply]
It says my max upload size is 1024.  I'm guessing that MB's?  
![maxupload](https://github.com/Guilouz/Creality-Helper-Script-Wiki/assets/94194418/56c1d6c9-48ce-4fce-87d7-7f0043c3b8e5)

**Reply by Guilouz** [reply]
Yes

**Reply by pitchforknumb** [reply]
Could time it takes to upload be an issue?  I don't have the fastest of Internet.

**Reply by Guilouz** [reply]
Yes it's possible timeout if your local network don't work as expected.

**Comment by martinrevert** [comment]
> Hey all. I've run into an issue where I can't upload a file from Orca Slicer that is larger than 100mb directly to my printer. It gives me a HTTP 502 error. I tried downloading the file to my desktop and uploading it in Fluidd and Mainsail that way, but both programs failed uploading. I then tried uploading it into Creality Print and it uploaded it just fine on that program. Is there a workaround for this where I can send the files straight from Orca Slicer? ![502 error](https://private-user-images.githubusercontent.com/94194418/327958870-01a20197-5ca5-439c-866a-3df244833f65.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU2NDcyNTYsIm5iZiI6MTcxNTY0Njk1NiwicGF0aCI6Ii85NDE5NDQxOC8zMjc5NTg4NzAtMDFhMjAxOTctNWNhNS00MzljLTg2NmEtM2RmMjQ0ODMzZjY1LmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MTQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTE0VDAwMzU1NlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTA1MzExNTFlM2FjNWM5ODU4Nzk3YTVlZmY3NjZkNjU4Zjg2NjViYjYyNWQyMDM5ODk3YjY4ZWNmMDhjYzFhYzAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.ZL0lINZK_a7TVBYuEKYKHpagtJnM25AlDI7ZPGGaHGg)

Hi @pitchforknumb , faced the same problem, had to export the gcode fron Orca and upload via SFTP to the gcodes dir over Linux. All other methods failed.

**Reply by Guilouz** [reply]
You must reinstall Moonraker or replace new service file here : https://github.com/Guilouz/Creality-Helper-Script/blob/main/files/services/S56moonraker_service

**Comment by Mr-Au** [comment]
I have a similar problem on K1 Max.  It error out whenever upload big files.  I then ssh into the machine and look at the disk usage, find out that the /tmp is 100%
My workaround is to upload large gcode via Creality's webUI, it will upload to /usr/data which have about 6.5G of space, more than enough for any big gcode files.
